docker-compose.yml

```diff
@@ -16,9 +16,10 @@ services:
       retries: 5
   web:
     build: .
-    command: /app/script/server
+    command: /app/script/server --development
     volumes:
       - .:/app
+      - node_modules:/app/cobrand_hackney/node_modules/
```

Comment:
Should the left hand side of this be cobrand_hackney/node_modules?

Reply:
It is intended to be a local Docker volume not a direct bind mount, so no.